sycl/source/detail/device_binary_image.cpp

```diff
@@ -120,9 +120,18 @@ void RTDeviceBinaryImage::print() const {
   std::cerr << "    Link options    : "
             << (Bin->LinkOptions ? Bin->LinkOptions : "NULL") << "\n";
   std::cerr << "    Entries  : ";
-  for (sycl_offload_entry EntriesIt = Bin->EntriesBegin;
-       EntriesIt != Bin->EntriesEnd; ++EntriesIt)
-    std::cerr << EntriesIt->name << " ";
+
+  sycl_offload_entry EntriesIt;
+  auto IncrementEntriesIt = [&]() {
```

Comment:
Can this be made a member function of sycl_offload_entry?

Thanks

Reply:
That's a good idea. Done in 9ec31163d97262a168cb522d1acb72a33b85c8ff